xml/System.Threading/Timer.xml

```diff
@@ -481,7 +481,7 @@ Sub TimerCallback(state As Object)
   
  ]]></format>
         </remarks>
-        <exception cref="T:System.ArgumentOutOfRangeException">The <paramref name="dueTime" /> or <paramref name="period" /> parameter is negative and is not equal to <see cref="F:System.Threading.Timeout.Infinite" />.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">The <paramref name="dueTime" /> or <paramref name="period" /> parameter is negative and is not equal to (-1).</exception>
```

Comment:
The original uses Timeout.Infinite; you changed it to -1.  Why?  Please revert this.

Reply:
This must be a bad regex or something I would almost swear I didn't change this yet it's somehow changed.